supported-tools.md

```diff
@@ -405,6 +405,8 @@ formatting.
   * [ibm_validator](https://github.com/IBM/openapi-validator)
   * [prettier](https://github.com/prettier/prettier)
   * [yamllint](https://yamllint.readthedocs.io/)
+* OpenSCAD
+  * [SCA2D](https://gitlab.com/bath_open_instrumentation_group/sca2d)
```

Comment:
Since this tool works with files on disk we need to add the :floppy_disk: emoji.

Reply:
No problem, sorry I missed that.